### ISSUE TITLE
Controls not visible in SStab

### ISSUE THREAD

**Issue by dmrvb** [twinbasic/twinbasic#1409]
**Describe the bug**
Most controls on an SStab are not visible at either design time or runtime.

**To Reproduce**
Create a simple project with an SStab (3 tabs) in VB6 with various controls on various tabs
Import into tB.
Most of the controls will not be visible in any tab.
I suspect that only one control will be visible but need to diagnose further.

**Expected behavior**
All controls should be visible at design and run time on their selected tabs..

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 11]
 - twinBASIC compiler version [201]

**Additional context**
I am surprised that nobody else has reported this but I don't think I am doing anything unusual.

**Comment by WaynePhillipsEA** [comment]
At the moment, ActiveX control containers aren't painting lightweight controls, so Image/Label/Shape can't be used inside them.

**Comment by dmrvb** [comment]
ok, thanks. Is it a lot of effort to get the SStab control fully working? Last time I looked I noted that RADBasic had decided not to implement this control.

**Comment by WaynePhillipsEA** [comment]
It will come together in the next couple of weeks as there is a lot of ongoing work for ActiveX controls

**Comment by camomille22003** [comment]
Same #1354

**Comment by dmrvb** [comment]
Just a gentle reminder that this bug is still outstanding. 
I use SSStab extensively so this is preventing me from going much further with tB testing. (there is also a run time error that I assume is related)

**Comment by dmrvb** [comment]
and another gentle reminder that SStab is still not working.
SStab, and the inability to load control array elements at run time is preventing any of my projects from importing/running. I've got some very complex GUI stuff so it might be good if I could do some testing on this...though it looks like Wayne is pretty busy already.

**Comment by WaynePhillipsEA** [comment]
Supporting loading of controls at runtime was expected to be added this week, but likely won't land until next week now.

**Comment by dmrvb** [comment]
There is no hurry but SStab and control loading would really help me as I would like to contribute a bit more to testing and bug reporting.

**Comment by fafalone** [comment]
This is sadly still an issue almost 1.5y later and happening on tB-originated projects, not just imports, and controls placed at design time, not just ones created at runtime (I didn't actually check those). There seems to be no relationship to where you create controls during design time and whether they'll show at all, and if they do, which tab they'll appear on.

Recommend taking a look at it because this is a quite commonly used, shipped-with-VB control and there's no tB replacement yet (there's VBCCR but that doesn't have the functionality of SSTab, which is why people still put up with that Win95-looking ancient thing).

**Comment by WaynePhillipsEA** [comment]
Yes, it's known.   It's waiting on some backend changes with regards to how the form designer works before it can be worked on.

**Comment by fafalone** [comment]
I think most SSTab issues are resolved now, including importing correctly; if there's any lingering problems create specific issues.
